tests/test_api_v1.py

```diff
@@ -612,15 +655,28 @@ def test_set_pkg_managers(app, client, db, worker_auth_env):
     assert get_rv.json["pkg_managers"] == ["gomod"]
 
 
-@pytest.mark.parametrize("bundle_exists", (True, False))
+@pytest.mark.parametrize(
+    "pkg_managers, bundle_exists", ((["gomod"], True), (["gomod"], False), (["npm"], True))
```

Comment:
Do we need the following scenarios?
* `['npm'], False`
* `['golang', 'npm'], False`
* `['golang', 'npm'], True`

Reply:
I'll use two parameterize decorators to cover all scenarios.